### PR TITLE
feat: New link in website header

### PR DESCRIPTION
**Pull request by nilubava**
https://github.com/prisma/homepage-v8/issues/315

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,7 +36,8 @@
     "prepublishOnly": "npm run build"
   },
   "dependencies": {
-    "feather-icons": "4.28.0"
+    "feather-icons": "4.28.0",
+    "react-feather": "^2.0.9"
```
Comment (madebysid) [inline package.json:40]: We already have an `Icon` component that exposes an API to render icons, can we use that?

(It's in a different bundle though, not sure if that'll cause problems)

**Comment by madebysid** [comment]
BTW I discussed this with Luan as well (I thought he was going to make these changes), since we're making significant changes to the header, why don't we take this opportunity to move the header to Tailwind like we've been planning?

**Comment by nilubava** [comment]
@madebysid Good idea to add the tailwind. I will also check out the other icon package.

**Comment by madebysid** [comment]
Can this PR be closed in favour of https://github.com/prisma/lens/pull/272?

**Comment by madebysid** [comment]
Is this PR stale now? Can we close it / delete the branch?

**Pull request closed without merging**